packages/components/src/components/hds/alert/title.hbs

```diff
@@ -2,8 +2,8 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: MPL-2.0
 }}
-{{! IMPORTANT: we removed any extra newlines before/after the `let` to reduce the issues with unexpected whitespaces (see https://github.com/hashicorp/design-system/pull/1652) }}
-{{#let (element this.componentTag) as |Tag|}}<Tag
-    class="hds-alert__title hds-typography-body-200 hds-font-weight-semibold"
-    ...attributes
-  >{{yield}}</Tag>{{/let}}
\ No newline at end of file
+{{~#let (element this.componentTag) as |Tag|}}
```

Comment:
[suggestion] I know @KristinLBradley suggested to use the `~` operator, and I know this is much more readable and "nice", but personally in this specific case I would revert this change: Ember has a weird way of handling whitespace, and the one you're using doesn't assure 100% that will remove it (see how it's used [in other parts of the codebase](https://github.com/search?q=%7E+user%3Ahashicorp+repo%3Ahashicorp%2Fdesign-system+language%3AHandlebars&type=code&ref=advsearch&l=Handlebars))

if y